src/twisted/names/hosts.py

```diff
@@ -87,21 +87,21 @@ def __init__(self, file=b"/etc/hosts", ttl=60 * 60):
 
     def _aRecords(self, name):
         """
-        Return a tuple of L{dns.RRHeader} instances for all of the IPv4
+        Return a list of L{dns.RRHeader} instances for all of the IPv4
```

Comment:
Why this change? More mutability seems worse. Could we add an annotation and call it a `Sequence`?